apiserver/worker/compiler.py

```diff
@@ -278,6 +278,10 @@ def compile(self, bot_dir, globs, errors, timelimit):
         ["gnatbind"],
         ["gnatlink", "-o", BOT],
     ],
+    "CMake": [
+        ["cmake", "."],
+        ["make", "-j2"],
```

Comment:
How many cores do workers get for compilation? If it's 1 I'll change this.

Reply:
It should have access to all 4 cores during compilation